src/liberty/devCommands.ts

```diff
@@ -490,28 +490,28 @@ export async function openReport(reportType: string, libProject?: LibertyProject
         if (path !== undefined) {
             let report: any;
             if (libProject.getContextValue() === LIBERTY_MAVEN_PROJECT || libProject.getContextValue() === LIBERTY_MAVEN_PROJECT_CONTAINER) {
-                report = Path.join(path, "target", "site", reportType + "-report.html");
+                report = getReportFile(path,"reports",reportType+".html");
             } else if (libProject.getContextValue() === LIBERTY_GRADLE_PROJECT || libProject.getContextValue() === LIBERTY_GRADLE_PROJECT_CONTAINER) {
                 report = await getGradleTestReport(libProject.path, path);
             }
             let reportTypeLabel = reportType;
             if (reportType === "gradle") {
                 reportTypeLabel = "test";
             }
-            fs.exists(report, (exists) => {
-                if (exists) {
-                    const panel = vscode.window.createWebviewPanel(
-                        reportType, // Identifies the type of the webview. Used internally
-                        libProject.getLabel() + " " + reportTypeLabel + " report", // Title of the panel displayed to the user
-                        vscode.ViewColumn.Two, // Open the panel in the second window
-                        {}, // Webview options
-                    );
-                    panel.webview.html = getReport(report); // display HTML content
-                } else {
-                    const message = localize("test.report.does.not.exist.run.test.first", report);
-                    vscode.window.showInformationMessage(message);
+            if(libProject.getContextValue() === LIBERTY_MAVEN_PROJECT || libProject.getContextValue() === LIBERTY_MAVEN_PROJECT_CONTAINER){
```

Comment:
Could you please add comments for the new lines of code? That would be helpful.

Reply:
added comments with the new commit , please [review](https://github.com/OpenLiberty/liberty-tools-vscode/pull/347/commits/e15319fc6708e0e9d74ef98debfa100e6680565b)